app/src/main/java/de/danoeh/antennapod/preferences/PreferenceController.java

```diff
@@ -232,8 +232,7 @@ private void setupInterfaceScreen() {
                         if (newValue.equals("page")) {
                             final Context context = ui.getActivity();
                             final String[] navTitles = context.getResources().getStringArray(R.array.back_button_go_to_pages);
-                            final String[] navTags = new String[3];
-                            System.arraycopy(MainActivity.NAV_DRAWER_TAGS, 0, navTags, 0, 3);
+                            final String[] navTags = { MainActivity.NAV_DRAWER_TAGS[0], MainActivity.NAV_DRAWER_TAGS[1], MainActivity.NAV_DRAWER_TAGS[2] };
```

Comment:
I think it would be better to directly reference the tags here. Relying on array indices might confuse future developers when changing `MainActivity.NAV_DRAWER_TAGS`. I suggest using `QueueFragment.TAG` etc.

Reply:
Another option would be to use a resource array just like for the titles. This way, when adding an item to the list, one only needs to change the resource file